src/shared/components/features-section/feature-item/index.module.css

```diff
@@ -0,0 +1,80 @@
+@import "shared/styles/imports/variables.css";
+@import "shared/styles/imports/custom-medias.css";
+
+.featureItem {
+    width: calc(20% - 5px);
+    margin-right: 5px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    &:last-child {
+        margin-right: unset;
+    }
+
+    & .image {
+        width: 70%;
+        margin-bottom: 15px;
+
+        & svg g {
+            &:hover {
+                cursor: pointer;
+
+                & path[id="back"] {
+                    fill: var(--color-yellow);
+                }
+            }
+
+            & path {
+                transition: 600ms ease;
```

Comment:
As you're using `seconds` in other transitions, you should use here too.

```sass
transition: 0.6s ease;
```

I also noticed it's the same value, so you should put it in a variable like `transition-duration`.

Reply:
I think that the one you saw using `seconds`, was the only one. So, I changed it to `ms`.